Scalar.Common/Git/GitProcess.cs

```diff
@@ -84,6 +84,26 @@ public GitProcess(string gitBinPath, string workingDirectoryRoot)
 
         public bool LowerPriority { get; set; }
 
+        public static Result Clone(string gitBinPath, string url, string targetDir, bool sparse, bool filterBlobs)
+        {
+            string sparseArg = sparse ? "--sparse " : string.Empty;
+            string filterArg = filterBlobs ? "--filter=blob:none" : string.Empty;
+
+            string dir = Paths.ConvertPathToGitFormat(targetDir);
+            dir.Replace(" ", "\\ ");
```

Comment:
Is the '\\' a Windows-ism?

Reply:
This is to escape spaced repo names, but this will not be valuable. Spaces need to be changed by the user to use `%20` at the top-level command.